src/webportal/src/app/job-submission/components/tools/job-ssh.jsx

```diff
@@ -138,20 +154,33 @@ export const JobSSH = ({
           content={`Choose SSH public key for job. Users should maintain the SSH private key themselves.`}
         />
       </Stack>
-      <Hint>
-        Enable Users SSH to allow user attach job containers through
-        corresponding ssh private key.
-      </Hint>
       <Toggle
-        label='Enable Job SSH'
+        label={
+          <div>
+            Enable Job SSH{' '}
```

Comment:
The names of two sections are confusing, maybe ```Enable Ssh between Containers``` is better?
BTW, in which scenario should user use this kind of ssh?

Reply:
SSH is must be enabled for MPI jobs. Other jobs can just ignore it.